whylog/tests/tests_client/test_files/001_most_basic/parsers.yaml

```diff
@@ -1,6 +1,10 @@
 ---
+class: RegexParser
+log_type : default
```

Comment:
I think this should be a default setting, so that whylog configuration is easier to comprehend for new users who only have one type of logs.